### DISCUSSION TITLE
MWM files from MAPS.ME app not useable here?

### DISCUSSION THREAD

**Discussion by Ere-the-Weekend-comes** [organicmaps/organicmaps#8872] [Q&A]
Downloaded 13 GB of MWM files this morning on Maps With Me then because of the amount of things I was dissatisfied with about the Maps.Me app, i searched for an alternative and found this app.

So I have these brand new downloaded maps, 200 are new, 245 in all, and I moved them in the correct location here, but they don't show anything (only remove the download option).  

Any way to get these to work?  I looked back thru the old topics till I had too many things in my head to remember all.  All the way back into mid 2023.  Page 8 of the topics, and didn't find this listed about the unusable maps me files.

**Comment by biodranik** [comment]
No way. Our maps have more improvements and details and are not compatible with maps.me, so you need to re-download them again.
